_faq/high_involvement_status.md

```diff
@@ -0,0 +1,5 @@
+---
+question: "What is the meaning of High Involvement status of any project?"
+---
+
+It simply means that the project already has more than one proposal and if you want to submit the proposal for the project, you should be confident enough that your proposal will be selected. Since we can't accept two proposals for the same project.
```

Comment:
break lines at 80cpl

"be confident enough" that is never possible.  Google may give us 2 slots only, in which case nobody can be confident of getting a slot.

"Since we can't accept two proposals for the same project." isnt a sentence.  something is missing.

Reply:
I think, I can use ''make sure that your proposal should have high chances of getting selected" instead of "be confident enough".